README_zh-CN.md

```diff
@@ -2,35 +2,35 @@
   <img src="https://github.com/open-mmlab/mmaction2/raw/main/resources/mmaction2_logo.png" width="600"/>
   <div>&nbsp;</div>
   <div align="center">
-    <b><font size="5">OpenMMLab 官网</font></b>
+    <b><font size="5">OpenMMLab官网</font></b>
```

Comment:
Thanks for your help. This is actually one of our conventions that there should be a whitespace between English and Chinese characters. Therefore there is no need for any modifications.

Reply:
Updated. add space back between the two